src/app/settings/reducer.ts

```diff
@@ -123,7 +130,9 @@ export const initialState: Settings = {
 
   language: defaultLanguage(),
 
-  colorA11y: '-'
+  colorA11y: '-',
+  wishListSource:
+    'https://raw.githubusercontent.com/48klocs/dim-wish-list-sources/master/voltron.txt'
```

Comment:
One thing I thought about - if someone has a wish list already loaded, will this override it?

Reply:
Basically, does this make it impossible to load a wishlist from a file? If so, should we remove the load-from-file capability?

Reply:
It does override the wish list that people had loaded previously the first time they load it up, but loading from a file still works.

When you load a file after that, `wishListSource`'s value is cleared out. I have separate analytics entries for fetch vs. load - fetch will be triggered a lot more, but we'll be able to see if anyone's loading file-based lists, and we can rip it out if not.